packages/mds-compliance-service/repository/entities/compliance-violation-entity.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BigintTransformer, IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { Column, Entity } from 'typeorm'
+import { ComplianceViolationDomainModel } from '../../@types'
+
+export interface ComplianceViolationEntityModel extends IdentityColumn, RecordedColumn {
+  violation_id: ComplianceViolationDomainModel['violation_id']
+  timestamp: ComplianceViolationDomainModel['timestamp']
+  policy_id: ComplianceViolationDomainModel['policy_id']
+  provider_id: ComplianceViolationDomainModel['provider_id']
+  rule_id: ComplianceViolationDomainModel['rule_id']
+  event_timestamp: ComplianceViolationDomainModel['violation_details']['event_timestamp']
+  device_id: ComplianceViolationDomainModel['violation_details']['device_id']
+  trip_id: ComplianceViolationDomainModel['violation_details']['trip_id']
```

Comment:
nit, I think we've started leaning towards defining these type explicitly instead of referencing the `DomainModel`, mostly due to some strange TypeORM inference that gets screwy when using referential types. (I can hunt through slack to find an example of this happening, but it was a huge nightmare to diagnose)

For example, `violation_id` would be an explicit `violation_id: UUID` instead. It's still totally safe for us to do this, because the mapper layers should enforce that domain & entity are compatible with the transforms done there.